### DISCUSSION TITLE
<None Include="README.md"/> not showing in project

### DISCUSSION THREAD

**Discussion by pkese** [ionide/ionide-vscode-fsharp#1582] [Q&A]
If I add this to `.fsproj` file,
```xml
<ItemGroup>
    <None Include="paket.references" />
    <None Include="README.md" />
</ItemGroup>
```
why won't these files appear in Ionide's solution explorer?

Should I add them in some other way?  
Is this documented anywhere?

**Comment by baronfel** [comment]
I believe we only add Compile and Content items to the display right now, but we'd have to dive into the implementation. The question would be:

* Are None items returned from proj-info, then
* Are they returned from the workspace notifications in FsAutoComplete, and 
* are they rendered as tree nodes in the Ionide Solution Explorer?
